frontend/src/components/explorer/gemBrowser/GemSearch.vue

```diff
@@ -102,25 +112,41 @@ export default {
       searchTermString: state => state.search.searchTermString,
     }),
     ...mapGetters({
+      models: 'models/models',
       searchResults: 'search/categorizedAndSortedResults',
     }),
     placeholder() {
-      if (this.metabolitesAndGenesOnly) {
-        return 'uracil, malate, SULT1A3, CNDP1';
-      }
       return 'uracil, SULT1A3, ATP => cAMP + PPi, subsystem or compartment';
     },
   },
+  async created() {
+    await this.getIntegratedModelList();
+  },
   mounted() {
-    if (this.searchTermString.length === 0) {
-      $('#search').focus();
-    } else {
-      this.blur();
-    }
+    $('#search').focus();
```

Comment:
This focus doesn't seem to work at the moment, might it be because of the transition?

Reply:
I'm looking into this now. Not sure what happened.

Reply:
I added a fix for it now. It must have been something I overlooked before so thanks for spotting it.